x-pack/plugins/features/common/feature_kibana_privileges.ts

```diff
@@ -263,4 +265,11 @@ export interface FeatureKibanaPrivileges {
    * @see UICapabilities
    */
   ui: readonly string[];
+
+  /**
+   * An optional list of other registered feature or sub-feature privileges that this privilege is composed of. When
+   * privilege is registered with Elasticsearch, it will be expanded to grant everything that referenced privileges
+   * grant. This property can only be set in the feature configuration overrides.
+   */
+  composedOf?: readonly FeatureKibanaPrivilegesReference[];
```

Comment:
__note:__ I didn't add `composedOf` and `disabled` to a feature schema to prevent consumers from using it during feature registration. Trying to contain the maintenance scope.